docs/tutorials/using-host-labels.md

```diff
@@ -0,0 +1,204 @@
+    type = webserver
+    location = us-seattle
+    provider = aws
+```
+
+Once you've written a few host labels, you need to enable them. Instead of restarting the entire Netdata service, you
+can reload labels using the helpful `netdatacli` tool:
+
+```bash
+netdatacli reload-labels
+```
+
+Your host labels will now be enabled. You can double-check these by using `curl http://HOST-IP:19999/api/v1/info` to
+read the status of your agent. For example, from a VPS system running Debian 10:
+
+```json
+{
+  ...
+	"host_labels": {
+	  "_is_master": "false",
+		"_virt_detection": "systemd-detect-virt",
+		"_container_detection": "none",
+		"_container": "unknown",
+		"_virtualization": "kvm",
+		"_architecture": "x86_64",
+		"_kernel_version": "4.19.0-6-amd64",
+		"_os_version": "10 (buster)",
+		"_os_name": "Debian GNU/Linux",
+		"type": "webserver",
+    "location": "seattle",
+    "provider": "aws"
+	},
+  ...
+}
+```
+
+You may have noticed a handful of labels that begin with an underscore (`_`). These are automatic labels.
+
+### Automatic labels
+
+When Netdata starts, it captures relevant information about the system and converts them into automatically-generated
+host labels. You can these use these to logically organize your systems via health entities, exporting metrics,
+streaming/master status, and more.
+
+They capture the following:
+
+-   Kernel version
+-   Operating system name and version
+-   CPU architecture, system cores, CPU frequency, RAM, and disk space
+-   If Netdata is running inside of a container
+    -   If so, OS and hardware details about the container's host
+-   What virtualization layer the system runs on top of, if any
+-   Whether the system is a streaming master or slave
+
+If you want to organize your systems without manually creating host tags, try the automatic labels in some of the
+features below.
+
+## Host labels in streaming
+
+You may have noticed the `_is_master` and `_is_slave` automatic labels from above. Host labels are also now streamed
+from a slave to its master agent, which concentrates an entire infrastructure's OS, hardware, container, and
+virtualization information in one place: the master.
+
+Now, if you'd like to remind yourself of how much RAM a certain slave system has, you can simply access
+`http://localhost:1999/host/SLAVE_NAME/api/v1/info` and reference the automatically-generated host labels from the slave
+system. It's a vastly simplified way of accessing critical information about your infrastructure.
+
+You can also use `_is_master`, `_is_slave`, and any other host labels in both health entities and metrics exporting.
+Speaking of which...
+
+## Host labels in health entities
+
+You can use host labels to logically organize your systems by their type, purpose, or location, and then apply specific
+alarms to them.
+
+For example, let's use configuration example from earlier:
+
+```conf
+[host labels]
+    type = webserver
+    location = us-seattle
+    provider = aws
```

Comment:
Only to confirm, I know "aws" can be understood as three letters only, but it is also the abbreviation for Amazon Web Services, that is also written as lowercase. Could this create a problem for us?

Reply:
I just changed this to `installed = 20200218` to mimic what you did in the health docs :smile: 

Better avoid any potential conflict—I just used that as a completely random example.

Reply:
All right :)